src/lib.rs

```diff
@@ -204,6 +203,18 @@ impl<P> Api<P>
         hexstr_to_u256(result_str).unwrap()
     }
 
+    pub fn get_finalized_head(&self) -> WsResult<String> {
```

Comment:
String result isn't very nice. can we decode this to `Header`? I guess we could use the generic header in `use sr_primitives::generic::Header;`

Reply:
```
let ret = api.get_header(None).unwrap();
let hdr: Header = serde_json::from_str(&ret).unwrap();
```